Makefile

```
@@ -14,7 +14,8 @@ YELLOW=\033[1;33m
 BLUE=\033[0;34m
 NC=\033[0m # No Color
 
-.PHONY: help build test test-coverage lint fmt run dev dev-setup dev-down install-tools security docker-build docker-run deploy clean
+.PHONY: help build test test-coverage lint fmt go-fmt run dev dev-setup dev-down install-tools security docker-build docker-run deploy clean \
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)